src/test/java/org/xstefank/whitelist/AddUserCommandTest.java

```diff
@@ -0,0 +1,70 @@
+package org.xstefank.whitelist;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.File;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.xstefank.ci.CILoader;
+import org.xstefank.model.PersistentList;
+import org.xstefank.model.Utils;
+import org.xstefank.TestUtils;
+import org.xstefank.api.GitHubAPI;
+
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CILoader.class, GitHubAPI.class})
+public class AddUserCommandTest {
+
+    private static final File userListFile = new File(TestUtils.TARGET_DIR, Utils.USERLIST_FILE_NAME);
+    private static final TestCI testCI = new TestCI();
+
+    private final PersistentList userList = new PersistentList(userListFile);
+    private final PersistentList adminList = new PersistentList(TestUtils.RESOURCES_PATH, Utils.ADMINLIST_FILE_NAME);
+    private AddUserCommand addUserCommand;
+
+    @BeforeClass
+    public static void beforeClass() {
+        TestUtils.deleteFileIfExists(userListFile);
+        PowerMockito.stub(method(CILoader.class, "getCI", String.class))
```

Comment:
I don't understand why `getCI` method needs to be stubbed, maybe it's a little too much :) would it be possible to stub method that calls this? if not than it's ok

Reply:
I think it makes sense because instead of this i would need to stub the loadCIs method that adds all CI to list which is used for triggering them.